src/main/java/com/starteo/demo/endpoint/controller/FundController.java

```diff
@@ -29,13 +29,21 @@ public class FundController {
   private AuthService authService;
 
   @PutMapping("/funds/{idea_id}")
-  public List<Fund> saveFundByIdea(@PathVariable(name = "idea_id")String ideaString, HttpServletRequest request, @RequestBody CreateFund toCreate) {
+  public List<Fund> saveFundByIdea(
+      @PathVariable(name = "idea_id") String ideaString,
+      HttpServletRequest request,
+      @RequestBody CreateFund toCreate) {
     User userConnected = authService.whoami(request);
-    com.starteo.demo.repository.model.Fund created = fundService.saveFundByIdeaId(toCreate, ideaString, userConnected);
+    com.starteo.demo.repository.model.Fund created =
+        fundService.saveFundByIdeaId(toCreate, ideaString, userConnected);
     return List.of(fundMapper.toRest(created));
   }
```

Comment:
When you provide the id in path it's for updating a specific funds. To create an idenpotant endpoint it's better to use directly the /funds endpoint and it's in the payload that we will add the id or not to create or update funds.